graphics/nvidia-drm-kmod/pkg-descr

```diff
@@ -1,9 +1,13 @@
-On FreeBSD 13.1-RELEASE and greater: this meta port will automatically install
-the recommended version of the port of Linux's nvidia-drm.ko kernel module for
-NVIDIA graphics hardware.
+FreeBSD port of Linux's nvidia-drm.ko Kernel module.
+
+graphics/nvidia-drm is an alternative to the x11/nvidia-driver series of ports.
```

Comment:
This is just straight up wrong. nvidia-drm.ko is a *part* of the NVIDIA driver, it's an addition to `x11/nvidia-driver` not a replacement for it.

Reply:
Thanks! I'll fix this (and the wiki).

Reply:
[`fbc810b` (#2)](https://github.com/grahamperrin/freebsd-ports/pull/2/commits/fbc810baf718fcfde16b64c713b7e81de26d618d) … better/correct?